implementations/rust/ockam/ockam_command/src/lib.rs

```diff
@@ -446,3 +448,62 @@ pub(crate) fn replace_hyphen_with_stdin(s: String) -> String {
         s
     }
 }
+
+const EXIT_OK: i32 = 0;
+const EXIT_USAGE: i32 = 2;
+
+fn show_help(help: clap::Error) {
+    paginate(&help.render()).unwrap_or_else(|_| help.print().unwrap());
+
+    let _ = std::io::stdout().lock().flush();
+    let _ = std::io::stderr().lock().flush();
+    process::exit(if help.use_stderr() {
+        EXIT_USAGE
+    } else {
+        EXIT_OK
+    });
+}
+
+fn paginate(text: &StyledStr) -> Result<()> {
+    let mut try_fallback = false;
+    let preferred_pager = std::env::var("PAGER").unwrap_or_else(|_| {
+        try_fallback = true;
+        "less".to_string()
+    });
+
+    if let Err(e) = paginate_with(preferred_pager.borrow(), text) {
+        if try_fallback {
+            paginate_with("more", text)
+        } else {
+            Err(e)
+        }
+    } else {
+        Ok(())
+    }
+}
+
+fn paginate_with(pager: &str, text: &StyledStr) -> Result<()> {
+    let mut invocation = process::Command::new(pager);
+
+    if Path::new(pager)
+        .file_name()
+        .map_or("", |s| s.to_str().unwrap_or(""))
+        == "less"
+    {
+        invocation.env("LESS", "-F");
```

Comment:
Although this work as expected, the default behavior now strips the colors from the output. We still need that.